extract_pov_edits/ingest_utils/process.py

```diff
@@ -0,0 +1,128 @@
+"""
+Copyright 2018 Google Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-------------------------------------------------------------------------------
+
+Process Edits
+
+Computes diff between Wikipedia revisions at sentence level and extracts POV rejects.
+
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+import re
+import nltk
+import mwparserfromhell
+
+COMPONENT_THERESHOLD = 10000
+CONTEXT_RANGE = 3
+
+
+def change_close_by(CONTEXT_RANGE, ind, changes):
+  length = len(changes) - 1
+  for inc in range(1, CONTEXT_RANGE+1):
+    if inc + ind < length and changes[inc + ind]:
+      return True
+    if ind - inc >= 0 and changes[ind - inc]:
+      return True
+  return False
+
+
+def diff(sents, cur_sents, rev_id):
+  """Computes diff at sentence level."""
+  sents_old = [s for s in cur_sents.keys()]
```

Comment:
this is just sents_old = cur_sents.keys()
I told you this the previous time, no? Seems familiar.

Reply:
Sorry! Fixed now.